provisioning/miniprov/hare_mp/main.py

```diff
@@ -647,6 +647,19 @@ def generate_support_bundle(args):
         if args.t:
             cmd.append('-t')
             cmd.append(args.t[0])
+        if args.duration:
+            logging.info("Time bound log collection for %s", args.duration)
```

Comment:
Are these log messages for debugging purposes? I fail to understand the need for the same.

Reply:
If I use `noop()`, this will force the application to exit immediately.
Which means that whenever the user specifies `--duration`, `--size-limit`,` --stacktrace` or whatever from the new requirements, the following things will happen:

1] No logs will be generated at all.
2] hare_setup will exit with 0 exit code.
That's why I added logs here for debugging purposes.